test/fixtures/worldwide/canada_organisations.json

```diff
@@ -20,7 +20,7 @@
s\">80 Elgin Street</span><br>\n<span class=\"locality\">Ottawa</span> <span class=\"region\">Ontario</span> <span class=\"postal-code\">K1P 5K7</span><br>\n<span class=\"country-name\">Canada</span>\n</p>\n\n\n        <div class=\"email-url-number\">\n            <p class=\"email\">\n              <span class=\"type\">Email</span>\n              <a class=\"email\" href=\"mailto:trevor.kerr@fco.gov.uk\">trevor.kerr@fco.gov.uk</a>\n            </p>\n            <p class=\"tel\">\n              <span class=\"type\">Office</span>\n              +1 613 364 6134\n            </p>\n        </div>\n\n\n    </div>\n  </div>\n</div>\n\n<h2 id=\"department-for-international-trade-dit\">Department for International Trade (DIT)</h2>\n<p><a href=\"https://www.gov.uk/government/world/organisations/department-for-international-trade-canada\">Department for International Trade</a> helps UK-based companies succeed in the global economy. We also help overseas companies bring their high-quality investment to the UK’s dynamic economy, acknowledged as Europe’s best place from which to succeed in global business.</p>\n\n<h2 id=\"public-holidays-in-2018\">Public holidays in 2018</h2>\n<p>On the following public holidays, our office will be closed for normal business.  Emergency consular assistance for British nationals is available by telephone.</p>\n\n<table>\n  <thead>\n    <tr>\n      <th>Date</th>\n      <th style=\"text-align: right\">Holiday</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <td>Monday 1 January</td>\n      <td style=\"text-align: right\">New Year’s Day</td>\n    </tr>\n    <tr>\n      <td>Monday 19 February</td>\n      <td style=\"text-align: right\">Family Day</td>\n    </tr>\n    <tr>\n      <td>Friday 30 March</td>\n      <td style=\"text-align: right\">Good Friday</td>\n    </tr>\n    <tr>\n      <td>Monday 2 April</td>\n      <td style=\"text-align: right\">Easter Monday</td>\n    </tr>\n    <tr>\n      <td>Monday 21 May</td>\n      <td style=\"text-align: right\">Victoria Day</td>\n    </tr>\n    <tr>\n      <td>Monday 2 July</td>\n      <td style=\"text-align: right\">in lieu of Canada Day</td>\n    </tr>\n    <tr>\n      <td>Monday 6 August</td>\n      <td style=\"text-align: right\">Civic Holiday</td>\n    </tr>\n    <tr>\n      <td>Monday 3 September</td>\n      <td style=\"text-align: right\">Labour Day</td>\n    </tr>\n    <tr>\n      <td>Monday 8 October</td>\n      <td style=\"text-align: right\">Thanksgiving</td>\n    </tr>\n    <tr>\n      <td>Monday 12 November</td>\n      <td style=\"text-align: right\">in lieu of Remembrance Day</td>\n    </tr>\n    <tr>\n      <td>Monday 24 December</td>\n      <td style=\"text-align: right\">Christmas Privilege Day</td>\n    </tr>\n    <tr>\n      <td>Tuesday 25 December</td>\n      <td style=\"text-align: right\">Christmas Day</td>\n    </tr>\n    <tr>\n      <td>Wednesday 26 December</td>\n      <td style=\"text-align: right\">Boxing Day</td>\n    </tr>\n  </tbody>\n</table>\n</div>",
```

Comment:
Could 

```html
<div class="govspeak">
```

in `access_and_opening_times` be 

```html
<div class="gem-c-govspeak govuk-govspeak">
```

so it would use the gem's Govspeak component?